### DISCUSSION TITLE
CP2K Release 9.1

### DISCUSSION THREAD

**Discussion by oschuett** [cp2k/cp2k#2605] [Team Posts]
Hi Everybody,

the end of 2021 is getting closer. We'll continue our tradition of making a release towards the end of December. So, please merge your changes and update the [version history](https://www.cp2k.org/version_history) in time.

At the moment I'm not aware of any blocking issues.

**Comment by hfp** [comment]
With DBCSR 2.2.0-rc0, it seems feasible to match CP2K's release schedule. However, perhaps we want to re-tag/scrub DBCSR's [milestone](https://github.com/cp2k/dbcsr/issues?q=is%3Aopen+is%3Aissue+milestone%3Av2.2.)?

**Comment by oschuett** [comment]
I've added a draft of the release note to the [version history](https://www.cp2k.org/version_history).

**Comment by oschuett** [comment]
The dashboard is now pretty green and I'm not aware of any blocking issues. So unless there are objections, I will tag the release tomorrow.
